src/rrt/2dplane/2dplane.hpp

```diff
@@ -6,6 +6,14 @@
 namespace RRT {
 
 /**
+ * Hash function for Eigen::Vector2d
+ */
+static size_t hash(Eigen::Vector2d state) {
+    return (((int)state.x() * 100) << ((int)log10(state.y()) + 3)) +
+           (int)state.y() * 100;
+}
```

Comment:
Unless there is a good reason to run your own hash function, it might be better to use the std::hash one. If you have boost, you can also use the hashcombine thing talked about over [here](https://github.com/RoboJackets/robocup-common/pull/12)